doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1260 @@
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_250_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE and tool support
+====================
+
+|no_changes_yet_note|
+
+Application development
+=======================
+
+This section provides detailed lists of changes to overarching SDK systems and components.
+
+Build system
+------------
+
+* Removed the ``CONFIG_MCUBOOT_IMAGE_VERSION`` Kconfig option in favor of using a dedicated :ref:`application VERSION file <zephyr:app-version-details>` to set the version.
+  You can alternatively set the version by using the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` Kconfig option, but using a :file:`VERSION` file is the recommended approach.
+
+* The |NCS| name and version is now displayed instead of the Zephyr version as the default boot banner when applications boot.
+  This can be customized in user applications.
+
+nRF Front-End Modules
+---------------------
+
+* Updated the name of the ``nrf21540_ek`` shield to ``nrf21540ek``.
+
+Working with nRF91 Series
+=========================
+
+* Added support for :ref:`nrf91_modem_trace_uart_snippet`.
+  Snippet is used for nRF91 modem tracing with the UART backend for the following applications and samples:
+
+  * :ref:`asset_tracker_v2`
+  * :ref:`serial_lte_modem`
+  * All samples that use nRF91 Series DK except for :ref:`slm_shell_sample`, :ref:`modem_trace_flash`, :ref:`modem_trace_backend_sample`.
+
+  For samples where the UART trace backend is enabled by default, the configuration is added to the sample overlays and project configuration.
+
+* The default board revision for nRF9160 DK has changed to v0.14.0.
+  See :ref:`nrf9160_board_revisions` for more details.
+
+Working with nRF52 Series
+=========================
+
+* :ref:`ug_nrf52_developing`:
+
+  * Updated the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option to support MCUboot bootloader in the direct-xip mode, and added the related documentation.
+    For details, see :ref:`mcuboot_direct_xip_mode`.
+
+Working with nRF53 Series
+=========================
+
+|no_changes_yet_note|
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
```

Comment:
@peknis as discussed, apart from the information regarding Sidewalk landing pages that are already in the ncs documentation, adding the release notes
@ktaborowski @wojciechpluszynski  FYI
```suggestion

Amazon Sidewalk
---------------

Starting from 2.5.0 release, Amazon Sidewalk is a part of the nRF Connect SDK.

* Added: 

  * nRF53 production support for Amazon Sidewalk, including Bluetooth LE, LoRa, and FSK.
  * New distribution model (Amazon Sidewalk is now merged with nRF Connect SDK).

* Updated:

  * Suspended external flash in the Sidewalk mode to reduce power consumption. 
    External flash is only available in the DFU mode.
  * Merged Amazon Sidewalk libraries into a unified library.
  * Adjusted the sensor monitoring app to change to initialization state upon reconnecting.
    This setting prevents the cloud application from being stuck when it is disconnected for a long time.
  * Deactivated the Bluetooth LE GATT Client to optimize the configuration.

* Removed an experimental devcontainer for a better user experience.

```